### ISSUE TITLE
ripe-atlas-tools is unable to run under python 3.10

### ISSUE THREAD

**Issue by frankiexyz** [RIPE-NCC/ripe-atlas-tools#229]
```
ripe-atlas probe-search  --country sg --status 1
Traceback (most recent call last):
  File "/home/frankie/.local/bin/ripe-atlas", line 8, in <module>
    from ripe.atlas.tools.commands.measure import Factory as BaseFactory
  File "/home/frankie/.local/lib/python3.10/site-packages/ripe/atlas/tools/commands/measure/__init__.py", line 20, in <module>
    from .ping import PingMeasureCommand
  File "/home/frankie/.local/lib/python3.10/site-packages/ripe/atlas/tools/commands/measure/ping.py", line 18, in <module>
    from ...helpers.validators import ArgumentType
  File "/home/frankie/.local/lib/python3.10/site-packages/ripe/atlas/tools/helpers/validators.py", line 25, in <module>
    from ..settings import aliases
  File "/home/frankie/.local/lib/python3.10/site-packages/ripe/atlas/tools/settings/__init__.py", line 287, in <module>
    conf = Configuration().get()
  File "/home/frankie/.local/lib/python3.10/site-packages/ripe/atlas/tools/settings/__init__.py", line 36, in get
    r = self.deep_update(r, custom)
  File "/home/frankie/.local/lib/python3.10/site-packages/ripe/atlas/tools/settings/__init__.py", line 46, in deep_update
    if isinstance(v, collections.Mapping):
AttributeError: module 'collections' has no attribute 'Mapping'
```

**Comment by frankiexyz** [comment]
I managed to find a workaround as follow as `Mapping` moved to `collections.abc` in 3.10
```
diff --git a/ripe/atlas/tools/settings/__init__.py b/ripe/atlas/tools/settings/__init__.py
index 0e7d515..f790106 100644
--- a/ripe/atlas/tools/settings/__init__.py
+++ b/ripe/atlas/tools/settings/__init__.py
@@ -13,7 +13,6 @@
 # You should have received a copy of the GNU General Public License
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.

-import collections
 import copy
 import os
 import re
@@ -44,7 +43,11 @@ class UserSettingsParser(object):
         Stolen from http://stackoverflow.com/questions/3232943/
         """
         for k, v in u.items():
-            if isinstance(v, collections.Mapping):
+            try:
+                from collections import Mapping
+            except ImportError:
+                from collections.abc import Mapping
+            if isinstance(v, Mapping):
                 r = cls.deep_update(d.get(k, {}), v)
                 d[k] = r
             else:
```

**Comment by chrisamin** [comment]
Fixed in v3.0.0
